app/controllers/api/v1/resources_controller.rb

```diff
@@ -1,9 +1,25 @@
-class Api::V1::ResourcesController < ApplicationController
+require 'pry'
+class Api::V1::ResourcesController < ApiController
   def index
     render json: Resource.all
   end
 
   def show
     render json: Resource.find(params[:id])
   end
-end
\ No newline at end of file
+
+  def create
+    resource = Resource.new(resource_params)
+    if resource.save
+      render json: { resource: resource }
+    else
+      render json: { error: resource.errors.full_messages }, status: :unprocessable_entity
+    end
+  end
```

Comment:
feel free to merge this, but you should add a serializer sooner rather than later for line 14